src/profile/components/Settings/Notifications/index.tsx

```diff
@@ -1,200 +1,130 @@
-import React, { Component, ContextType } from 'react';
+import React, { useEffect, useCallback, useState } from 'react';
 
 import { md } from 'common/components/Markdown';
 import { Pane } from 'common/components/Panes';
-import {
-  getNotificationPermission,
-  resolveNotificationPermission,
-  verifyNotification,
-} from 'common/utils/notification';
-import {
-  getNotificationSubscription,
-  registerPushManager,
-  removeNotificationSubscription,
-  verifyPushManager,
-} from 'common/utils/pushManager';
-
-import { UserContext } from 'authentication/providers/UserProvider';
-import { getAllChannels, getUserChannels, postUserChannels, subscribe, unsubscribe } from 'profile/api/notifications';
+import { verifyNotification } from 'common/utils/notification';
+import { verifyPushManager } from 'common/utils/pushManager';
+
 import { verifyServiceWorker } from 'serviceworker/browser';
 
-import { IChannel } from '../../../models/Notification';
 import BrowserSupport from './BrowserSupport';
 import style from './notifications.less';
 import Option from './Option';
+import { useSelector, useDispatch } from 'core/redux/hooks';
+import { shallowEqual } from 'react-redux';
+import {
+  toggleDevicePermission,
+  registerDeviceForPushNotifications,
+  unregisterDeviceFromPushNotfications,
+  resolveCurrentDeviceSubscription,
+} from 'notifications/slices/subscriptions';
+import { Permissions } from './Permissions';
+import { fetchPermissions } from 'notifications/slices/permissions';
+import { fetchUserPermissions } from 'notifications/slices/userPermissions';
+import { __CLIENT__ } from 'common/constants/environment';
 
 const ABOUT_NOTIFICATIONS = md`
-  # Notifikasjoner
-
-  Notifikasjoner kan brukes til å sende deg varsler for hendelser som skjer
-  på nettsiden uten at du trenger å være på den direkte.
+  # Varsler
 
-  ## Denne funksjonaliteten har ikke blitt ferdigimplementert!
+  Varsler er alle henvendelser du kan motta på via både e-post og pushvarsler fra Onlines nettsider.
+  Du kan selv velge hvilke varsler du ønsker å motta, enten som e-post eller via push.
 `;
 
 const ABOUT_BROWSER_SUPPORT = md`
-  ### Nødvendige funksjoner
+  ### Nettlesersøtte
 
-  For at det skal være mulig å bruke notifikasjoner fullt ut
-  må følgende funksjonaliteter være tilgjengelige i nettleseren:
-`;
+  _Pushvarsler støttes foreløpig ikke på iPhone og iPad, fordi Apple har valgt å ikke implementere det._
```

Comment:
Yeah, probably better.
I just wanted to make it clear that it is not up to us that they are not supported. It is Apple that decided not to implement the open web standard for notifications because they want us to make an app to do that (which means we have to pay Apple). Notifications have been implemented in Safari and are working on Mac, they have just disabled it for iOS.